.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest a new algorithm, enhancement to an existing algorithm, etc.
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!--
+If you want to propose a new algorithm, please refer first to the scikit-learn
+inclusion criterion:
+https://scikit-learn.org/stable/faq.html#what-are-the-inclusion-criteria-for-new-algorithms
+-->
+
+#### Is your feature request related to a problem? Please describe
+
+#### Describe your proposed solution
+
+#### Describe alternatives you've considered
```

Comment:
```suggestion
#### Describe alternatives you've considered, if relevant
```

I'm not sure we actually want this section. IMO we don't want them to come up with other solutions just for the sake of it